keyboards/splitkb/kyria/keymaps/default/keymap.c

```diff
@@ -37,136 +46,171 @@ enum layers { _QWERTY = 0, _DVORAK, _COLEMAK_DH, _NAV, _SYM, _FUNCTION, _ADJUST
 // produces the key `tap` when tapped (i.e. pressed and released).
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
Just so it doesn't happen again...
```suggestion
// clang-format off
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```